### PR TITLE
Bump sbt to 1.5.7

### PR DESCRIPTION
**Pull request by jonathonherbert**
Co-authored-by: @andrew-nowak

## What does this change?

Bumps sbt to 1.5.7. Updates our sbt syntax along the  way to avoid deprecation warnings.

## How to test

Run locally or deploy to CODE. The project should build and run as before.

### CODE REVIEW

**Comment by jonathonherbert** [comment]
Tested on CODE – app loads and creates tags. What more could you ask of tag manager?

File: build.sbt [hunk 1]
```diff
@@ -63,9 +63,9 @@ lazy val root = (project in file(".")).enablePlugins(PlayScala, RiffRaffArtifact
     packageSummary := "tag manager",
     packageDescription := """manage tags""",
 
-    doc in Compile := (target.value / "none"),
+    Compile / doc := (target.value / "none"),
     scalaVersion := "2.12.13",
-    scalaVersion in ThisBuild := "2.12.13",
+    ThisBuild / scalaVersion := "2.12.13",
```
Comment (mchv) [inline build.sbt:68]: Just mentioning you should plan to update `scala` to `2.13.5` which should be quite straightforward.

Comment (jonathonherbert) [reply build.sbt:68]: Good to know. Looks like we'll need a 2.13 build for `panda-hmac-play` in consuming repositories, including this one.

**Review by rhystmills — APPROVED** [review]
Application looks to be running as expected locally and on CODE, and tests are all passing 👍

**Pull request merged**